ansible/roles/software/ckan/inventory/handlers/main.yml

```diff
@@ -3,8 +3,13 @@
   service: name=apache2 state=restarted
   when: datagov_in_service
 
-- name: reload supervisor
-  command: supervisorctl reload
+- name: reread supervisor
+  command: supervisorctl reread
```

Comment:
In your catalog-app PR you're calling reread && update. Don't you also want to do that here? For example, since we're renaming the supervisor process gunicorn-web-app to ckan, we want supervisor to apply those changes (removing gunicorn-web-app and adding ckan). I though the update command was necessary to apply it.